examples/tutorials/self-paced-training/part-3_security_and_privacy/chapter-6_Security_in_federated_compute_system/06.2_identity_security/identity_security.ipynb

```diff
@@ -0,0 +1,210 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# **Indentity Security**  \n",
+    ">note\n",
+    " This is section is the same as \n",
+    "[FLARE's Security documentation](https://nvflare.readthedocs.io/en/2.4/user_guide/security/identity_security.html)\n",
+    "\n",
+    "\n",
+    "## Authentication\n",
+    "\n",
+    "NVFLARE’s authentication model is based on Public Key Infrastructure (PKI) technology\n",
+    "\n",
+    "For the FL project, the Project Admin uses the Provisioning Tool to create a Root CA with a self-signed root certificate. This Root CA will be used to issue all other certs needed by communicating parties.\n",
+    "\n",
+    "Identities involved in the project (Server(s), Clients, the Overseer, Users) are provisioned with the Provisioning Tool. Each identity is defined with a unique common name. For each identity, the Provisioning Tool generates a separate password-protected Startup Kit, which includes security credentials for mutual TLS authentication:\n",
```

Comment:
"with the provision tool" -> "using the provision tool", it might be good adding a link to the doc of provision tool